src/main/java/service/impl/ReportWriterImpl.java

```diff
@@ -0,0 +1,21 @@
+package service.impl;
+
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import service.ReportWriter;
+
+public class ReportWriterImpl implements ReportWriter {
+
+    @Override
+    public void writeReport(String report, String toFileName) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+            writer.write(report);
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Cannot find " + toFileName + " file", e);
```

Comment:
Nice to see that you're not ignoring the FileNotFoundException and are throwing a RuntimeException instead. Remember that it's important to always handle exceptions properly instead of just ignoring them.